jujugui/static/gui/src/app/components/search-results/test-search-results.js

```diff
@@ -25,12 +25,18 @@ chai.config.includeStack = true;
 chai.config.truncateThreshold = 0;
 
 describe('SearchResults', function() {
+  var searchResults;
 
   beforeAll(function(done) {
-    // By loading this file it adds the component to the juju components.
+    // By loading these files it makes their classes available in the tests.
     YUI().use('search-results', function() { done(); });
   });
 
+  beforeEach(function() {
+    searchResults = new juju.components.SearchResults();
```

Comment:
This means that every component is 'rendered' and then interacted with vs testing specific methods directly.